src/cli/commands/build/build.ts

```diff
@@ -37,6 +40,15 @@ export async function build(options: SWACLIConfig) {
   if (options.auto && hasBuildOptionsDefined(options)) {
     logger.error(`You can't use the --auto option when you have defined appBuildCommand or apiBuildCommand in ${swaCliConfigFilename}`);
     logger.error(`or with the --app-build-command and --api-build-command options.`, true);
+    const cmdEndTime = new Date().getTime();
+
+    collectTelemetryEvent(TELEMETRY_EVENTS.Build, {
+      duration: (cmdEndTime - cmdStartTime).toString(),
+      flagsUsed: flagsUsedStr,
+      responseType: "PreConditionFailure",
```

Comment:
PreConditionFailure
BuildFailure

These strings are getting repeated across different files.. Shall we move these to constants.ts file?